lab_1_keywords_tfidf/main.py

```diff
@@ -137,7 +247,23 @@ def calculate_chi_values(expected: dict[str, float], observed: dict[str, int]) -
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(expected, dict) and isinstance(observed, dict):
```

Comment:
revert every condition and return None immediately

Reply:
done